packages/stack-shared/src/utils/urls.tsx

```diff
@@ -12,8 +13,8 @@ export function isValidUrl(url: string) {
   return !!createUrlIfValid(url);
 }
 
-export function isLocalhost(urlOrString: string | URL) {
-  const url = createUrlIfValid(urlOrString);
+export function isLocalhost(urlOrString: URL | string | null | undefined) { ;
+  const url = typeof urlOrString === "string" ? createUrlIfValid(urlOrString) : urlOrString;
```

Comment:
why this change?

Reply:
reverted it to its original implementation, my branch had a broken implementation for some reason before